### PR TITLE
ref(sampling): Add analytic keys for the new dynamic sampling experience - [TET-500]

### PR DESCRIPTION
**Pull request by priscilawebdev**
**What this PR does:**

- Add analytic keys for the new dynamic sampling experience
- Update the UI with these new keys

ps: the old samplingAnalyticsEvents will be removed when the LA  program ends

### CODE REVIEW

File: static/app/views/settings/project/dynamicSampling/dynamicSampling.tsx [hunk 1]
```diff
@@ -65,12 +65,15 @@ export function DynamicSampling({project}: Props) {
...
-    });
+    const biasIndex = biases.findIndex(b => b.id === type);
+
+    if (biasIndex === -1) {
+      return;
+    }
+
+    const newDynamicSamplingBiases = [...biases];
+    newDynamicSamplingBiases[biasIndex].active =
+      !newDynamicSamplingBiases[biasIndex].active;
```
Comment (matejminar) [inline static/app/views/settings/project/dynamicSampling/dynamicSampling.tsx:76]: Isn't this mutating the original object?
I think `[...biases]` does not create a deep clone.
Perhaps your previous implementation with map and `{...bias}` was better in this regard?

Comment (priscilawebdev) [reply static/app/views/settings/project/dynamicSampling/dynamicSampling.tsx:76]: yes... I still need to manually test these changes, but I can use the `lodash/deepClone` here

Comment (matejminar) [reply static/app/views/settings/project/dynamicSampling/dynamicSampling.tsx:76]: yeah, that would work, but honestly, I liked your map solution 👍

Comment (priscilawebdev) [reply static/app/views/settings/project/dynamicSampling/dynamicSampling.tsx:76]: yes... it was nice, but I needed to get the rule for this new logic https://github.com/getsentry/sentry/blob/0ad561f1c1ae61368ee1d92f00ec498f644f8da3/static/app/views/settings/project/dynamicSampling/dynamicSampling.tsx#L90 and I did not want to "loop" twice...but I have another idea and will update the code 😉

**Review by matejminar — APPROVED** [review]

**Pull request merged**